src/WebJobs.Extensions.DurableTask/EntityScheduler/Entity.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Statically accessible context for entity operations.
+    /// </summary>
+    public static class Entity
+    {
+        private static readonly AsyncLocal<IDurableEntityContext> EntityContext
+            = new AsyncLocal<IDurableEntityContext>();
+
+        /// <summary>
+        /// The context of the currently executing entity.
+        /// </summary>
+        public static IDurableEntityContext Context => EntityContext.Value;
+
+        /// <summary>
+        /// The key of the currently executing entity.
+        /// </summary>
+        public static string Key => EntityContext.Value.Key;
```

Comment:
Since `Key` and `Self` are already defined in `IDurableEntityContext`, are we really saving much by adding these extra properties? I almost wonder if code is actually more readable if written as `EntityContext.Current.Key` and `EntityContext.Current.Id` instead of `EntityContext.Key` and `EntityContext.Self`.